### ISSUE TITLE
Can't paste with `paste dest=` when `dest` has spaces

### ISSUE THREAD

**Issue by Gregaras** [ranger/ranger#2470]
#### Runtime Environment

- Ranger version/commit: ranger-master v1.9.3-422-g1de6787c

#### Current Behavior

When I specify path that has space characters in the `paste dest=` command and execute it, Ranger says "Failed to paste. Destination is invalid". If I enclose the path with " or ' it won't work. Getting rid of space characters solves the issue.

Also, same problem occurs with bookmarked paths and with things like `%d`, `%s`.

I think the program cuts the `dest` argument off at the part where first space is.

#### Expected Behavior

Able to paste when specifying path with spaces in `paste dest=` command.

#### Steps to reproduce
1. Make a folder in home folder called "test test"
2. Make a file in home folder called "test"
3. Cut "test" file
4. Try to paste it by executing command `paste dest=test test` or `paste dest='test test'` or `paste dest="test test"`

**Comment by Gregaras** [comment]
> or you can use eval, `:eval self.fm.paste(dest="test test")` should work.

Is it possible to make this work with bookmarked directory? I added a map like this: `map pt<any> eval self.fm.paste(dest=%any_path)` but it won't work.

**Comment by toonn** [comment]
Yeah, like I said, pasting from such a directory works. But you can map the command, `nmap pq eval self.fm.paste(dest="%f")` should work.
